cfme/tests/infrastructure/test_timelines.py

```diff
@@ -1,242 +1,177 @@
-# -*- coding: utf-8 -*-
-import fauxfactory
 import pytest
 
-from cfme.base.ui import Server
-from cfme.common.provider import BaseProvider
 from cfme.common.vm import VM
-from cfme.infrastructure.provider import InfraProvider
-from cfme.infrastructure.provider.scvmm import SCVMMProvider
-from cfme.rest.gen_data import a_provider as _a_provider
-from cfme.rest.gen_data import vm as _vm
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.openstack_infra import OpenstackInfraProvider
 from cfme.utils.appliance.implementations.ui import navigate_to
-from cfme.utils.blockers import BZ
 from cfme.utils.generators import random_vm_name
 from cfme.utils.log import logger
-from cfme.utils.providers import ProviderFilter
 from cfme.utils.wait import wait_for
-from fixtures.provider import setup_one_or_skip
 
 
-pytestmark = [pytest.mark.tier(2)]
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([VMwareProvider, RHEVMProvider, OpenstackInfraProvider], scope='module')
```

Comment:
REQUIRED: please restore the logic from a_provider where any infra provider except SCVMMProvider will be picked up.